server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetricPopup.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import BubblePopup from '../../../../components/common/BubblePopup';
+import MultiSelect from '../../../../components/common/MultiSelect';
+import { translate, translateWithParameters } from '../../../../helpers/l10n';
+
+interface Props {
+  elements: string[];
+  metricsTypeFilter: string[];
+  onSearch: (query: string) => Promise<void>;
+  onSelect: (item: string) => void;
+  onUnselect: (item: string) => void;
+  popupPosition?: any;
+  renderLabel: (element: string) => React.ReactNode;
+  selectedElements: string[];
+}
+
+export default function AddGraphMetricPopup(props: Props) {
+  let footerNode: React.ReactNode = '';
+  let displayFooterNode: boolean = false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type boolean trivially inferred from a boolean literal, remove type annotation ([typescript:S3257](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3257))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3149&issues=AWIpOFggnhKRZuqhdsGh&open=AWIpOFggnhKRZuqhdsGh)

[ISSUE_KEY]: AWIpOFggnhKRZuqhdsGh